src/main/java/core/basesyntax/figures/RightTriangle.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Figure;
+
+public class RightTriangle extends Figure {
+    private static final String type = "triangle";
+    private final int firstLeg;
+    private final int secondLeg;
+
+    public RightTriangle(int firstLeg, int secondLeg, String color) {
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+        this.color = color;
```

Comment:
The 'color' field is not defined in the class. You need to either add a color field in this class or in the parent class 'Figure' and make sure it is accessible here.